### ISSUE TITLE
Fixed-size types without length prefix?

### ISSUE THREAD

**Issue by jjl** [zombodb/pgx#143]
Hi, pgx looks cool!

I was just wondering about the ability to create new fixed-size datatypes that don't have a length prefix, i.e. that aren't just wrapping a variable length binary. basically just large integers. Is that something that's doable or that you've considered?

**Comment by eeeebbbbrrrr** [comment]
Yeah we do but there’s not a lot of help from pgx. You’ll have to write all the various functions yourself too. 

There’s a little test that outlines the basics. It’s a rough rust version of the Complex type described here:  https://www.postgresql.org/docs/13/xtypes.html

see: https://github.com/zombodb/pgx/blob/eec5e2f720236ede1eb2f106a8641623071d0ffa/pgx-tests/src/tests/struct_type_tests.rs

**Comment by pksunkara** [comment]
I think this needs to be categorized as a feature request instead of question.
